Mage.Sets/src/mage/cards/w/WinterthornBlessing.java

```diff
@@ -0,0 +1,84 @@
+package mage.cards.w;
+
+import mage.abilities.Ability;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.DontUntapInControllersNextUntapStepTargetEffect;
+import mage.abilities.effects.common.counter.AddCountersTargetEffect;
+import mage.abilities.keyword.FlashbackAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.counters.CounterType;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.TargetPermanent;
+import mage.target.common.TargetControlledCreaturePermanent;
+import mage.target.targetpointer.FirstTargetPointer;
+import mage.target.targetpointer.FixedTarget;
+import mage.target.targetpointer.SecondTargetPointer;
+
+import java.util.UUID;
+
+/**
+ * @author LePwnerer
+ */
+public final class WinterthornBlessing extends CardImpl {
+
+    public WinterthornBlessing(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{G}{U}");
+
+        // Put a +1/+1 counter on up to one target creature you control. Tap up to one target creature you don't control, and that creature doesn't untap during its controller's next untap step.
+        this.getSpellAbility().addTarget(new TargetControlledCreaturePermanent(0, 1));
+        this.getSpellAbility().addTarget(new TargetPermanent(0, 1, StaticFilters.FILTER_CREATURE_YOU_DONT_CONTROL));
+        this.getSpellAbility().addEffect(new AddCountersTargetEffect(CounterType.P1P1.createInstance()).setTargetPointer(new FirstTargetPointer()));
+        this.getSpellAbility().addEffect(new WinterthornBlessingEffect().setTargetPointer(new SecondTargetPointer()));
```

Comment:
If you're using `SecondTargetPointer` here then use `TapTargetEffect` and `DontUntapInControllersNextUntapStepTargetEffect` here too, otherwise you don't need it